tests/aio/test_strategy.py

```diff
@@ -114,6 +119,185 @@ async def test_fixed_window_with_elastic_expiry_multiple_cost(
         ).reset_time == pytest.approx(end + 2, 1e-2)
         assert not await limiter.hit(limit, "k2", cost=6)
 
+    @async_sliding_window_counter_storage
+    @async_fixed_start
+    async def test_sliding_window_counter(self, uri, args, fixture):
```

Comment:
Please also add a test case for the `test()` strategy method for sliding window strategy.